src/utils/getHyperLinkFromPath.ts

```diff
@@ -1,16 +1,14 @@
-/**
- * generates the clickable link to a specific folder
- * This will recieve the path of a specific folder
- * Gets the number of '/' chars in the path
- * If there is no '/' then nothing changes since the path is same as deepestDirName
- * If there are '/'s then it sets the name of the folder as the deepestDirName
- * Makes a const name hyperLink put the path and deepestDirName inside the github hyperlink format
- * returns the hyperLink
+/** @function getHyperLinkFromPath  generates the clickable link to a specific folder
+ * @param {string} path  The path of a specific folder like "./src/components/reusable"
+ * @returns {string} the hyperLink like [reusable](./src/components/reusable)
  */
 
 export const getHyperLinkFromPath = (path: string): string => {
   const curDepth = path.match(/\//g)?.length ?? 0;
+  /** Checks to see how many '/' is included in the address */
   const deepestDirName = curDepth ? deepestDirName : path;
```

Comment:
isn't 0 true? won't this always be true?

Reply:
I think 0 is false

Reply:
I tried this piece of code inside index.ts
    const curDepth = path.match(/\//g)?.length ?? 0;
    const a = curDepth ? "true" : "false";
    console.log(path,a);
this is what I get:

public false
index.ts:59 src false
index.ts:59 src/components true
index.ts:59 src/components/reusable true
index.ts:59 src/images true
index.ts:59 src/tree true
index.ts:59 src/utils true

Reply:
This line just doesn't make sense. what is deepestDirName = deepestDirName?

Reply:
sorry! my bad!